<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderSettings</class>
 <widget class="QDialog" name="RenderSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Render Settings</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>201</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Settings</string>
   </property>
   <widget class="QCheckBox" name="MSAABox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable MSAA</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DepthTestBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Depth Test</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="globLightBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Global Light</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="lightBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Lighting</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="colMaterialBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Color Material</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
